UK_Reports/reports/statement_of_account_report.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+	<data>
+		<template id="statement_of_account_body">
+			<t t-set="doc_header_string" t-value="'Statement of Account'"/>
+			<t t-call="UK_Reports.template_document_title"/>
+			<div class="row">
+				<div class="col-xs-7">
+					<strong>Statement To:</strong>
+					<br/>
+                    <span t-raw="o.contact_address.replace('\n\n', '\n').replace('\n', '&lt;br&gt;')"/>
+				</div>
+				<div class="col-xs-5">
+					<strong>Date</strong>
+					<span t-esc="context_timestamp(datetime.datetime.now()).strftime('%d-%m-%Y')"/>   
+					<br/>
+					<t t-if="o.ref">
+						<strong>Customer Ref</strong>
+						<span t-field="o.ref"/>
+						<br/>
+					</t>
+					<t t-if="o.user_id">
+						<strong>Account Manager</strong>
+						<span t-field="o.user_id.name"/>
```

Comment:
Is this always going to be right? ie from the account manager group

Reply:
Narender said yes